canvas/templates/canvas/event_create.html

```diff
@@ -0,0 +1,47 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block script %}
+    <script src="{% static 'canvas/js/create_event.js' %}"></script>
+{% endblock %}
+
+{% block header %}
+    Create Event for {{ course.name }}
+{% endblock %}
+
+{% block content %}
+    <form method="post" class="" style="align-content: left;">
+        {% csrf_token %}
+        {% for field in form %}
+            <div class="form-group">
+                {% if field.label %}
+                    <h6 class="form-label">
+                        {{ field.label_tag }}
+                    </h6>
+                {% endif %}
+                {% if field.errors %}
+                    {% for error in field.errors %}
+                        <div class="alert alert-danger alert-dismissible fade show" role="alert">
+                            {{ error }}
+                            <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+                                <span aria-hidden="true">&times;</span>
+                            </button>
+                        </div>
+                    {% endfor %}
+                {% endif %}
+                {{ field }}
+                {% if field.help_text %}
+                    <small class="form-text text-muted">{{ field.help_text }}</small>
+                {% endif %}
+            </div>
+        {% endfor %}
+        <input class="btn btn-success mt-md-5" type="submit" value="Submit">
+    </form>
+    <!-- XDSoft DateTimePicker -->
+    <link rel="stylesheet"
```

Comment:
These links should be in the script block. Also, you need to download the files, put it in the static folder and use {% static %} tag to load them in the script block.